sunpy/image/tests/test_slit.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import, division, print_function
+
+from sunpy.image import slit
+import sunpy.map
+import sunpy.data.test
+import astropy.units as u
+import numpy as np
+import os
+import pytest
+from glob import glob
+
+@pytest.fixture
+def eit_test_cube():
+    testpath = sunpy.data.test.rootdir
+    eit_dir = os.path.join(testpath, 'EIT')
+    return sunpy.map.Map(eit_dir, cube=True)
+
+@pytest.fixture
+def numbered_array():
+    test_numb = np.linspace(1,10,10)*np.ones((10,10))
+    return test_numb
+
+@pytest.fixture
+def e_list():
+    alist=[]
+    for i in range(1,11):
+        alist.append([i,i])
+    return alist
+
+
+def test_slit():
```

Comment:
when using fixtures the secret trick is to pass them in as arguments to the test functions:

```python
def test_slit(eit_test_cube):
    test = slit.slit(eit_test_cube,...)
```
pytest handles calling the function and returning the result for you.